src/lib/util/Config.ts

```diff
@@ -264,26 +266,35 @@ export class Config {
 	}
 
 	private async initializeConfig(): Promise<void> {
+		// TODO: This method's current functionality is because we need to support v2 and v3 at the same time.
+		//  Once we no longer need to support v2, we can change this method to, for example, not use a separate config file
+		//  for v3.
 		this.logger.trace(`Initializing Config`);
+		// If there's no v2-based config, we need to create one, so the customer can safely downgrade to v2 if they want.
+		// NOTE: When v3 becomes the primary version, we may want to delete this, and possibly rework the config files
+		// in general.
+		if (!await this.fileHandler.exists(this.legacyConfigFilePath)) {
```

Comment:
If a user is directly getting the `latest-pilot` and have never had 2.x, would it still be necessary to have the 2.x config version?

Reply:
I guess technically not, since it would be created if they ever downgrade to v2. At the time, it felt like doing it this way made the code cleaner, but I'll see whether the other way is easier.

Reply:
Actually, the way you proposed is correct. My way would instantiate a v3 config as the v2 config on a fresh install, which is bad. Expect this change shortly.